docs/Makefile

```
@@ -10,7 +10,7 @@ VERSION ?= $(shell echo $(TAGGED_VERSION) | cut -c 2-)
 
 HUGO_VERSION := 0.69.2
 
-SOLO_HUGO_THEME_REVISION := 41587dc9dc719bd22cedd9803994f756e25f571f
+SOLO_HUGO_THEME_REVISION := $(curl --silent "https://api.github.com/repos/solo-io/hugo-theme-soloio/releases/latest" | jq -r .tag_name)
```

Let's change this to 
```
curl --silent “https://api.github.com/repos/solo-io/hugo-theme-soloio/releases/latest” | jq -esr ‘if . == [] then null else .[] | .tag_name end’
```

else if the `curl` fails, jq will swallow the error and silently fail here.